### PR TITLE
Helm improvements (affinity, memory, healthcheck)

### PR DESCRIPTION
**Pull request by ioanaciont**

### CODE REVIEW

File: helm/notifications-push/values.yaml [hunk 1]
```diff
@@ -9,6 +9,8 @@ image:
   repository: coco/notifications-push
   pullPolicy: IfNotPresent
 resources:
+  requests:
+    memory: 100Mi
```
Comment (peteclark-ft) [inline helm/notifications-push/values.yaml]: This requests limit could probably be lower - `notifications-push` seems to run consistently at ~40Mi and spikes to ~100Mi, so I think the requests could be set to around the 50Mi mark? What do you think?

Comment (peteclark-ft) [reply helm/notifications-push/values.yaml]: Similarly, the `list-notifications-push` limits could be lower

Comment (ioanaciont) [reply helm/notifications-push/values.yaml]: I had another look in Grafana chart, and I think you are right - I'll change it to 50Mi.

Comment (tosan88) [reply helm/notifications-push/values.yaml]: Hi @peteclark-ft @ioanaciont ! In fact, what we have found out, that sometimes the grafana board doesn't show the exact metric values unless zoomed in to the peak. We had an OOM in K8s PROD for notifications-push as well so I would like to request to increase the memory both for limit and request ^^

**Pull request merged**